shared/src/persistence/dynamsoft/getScannerInterface.test.js

```diff
@@ -178,6 +182,53 @@ describe('getScannerInterface', () => {
     expect(error).toBeDefined();
   });
 
+  it('can enable duplex mode by calling startScanSession with scanMode set to `DUPLEX`', async () => {
+    const scannerAPI = getScannerInterface();
+    scannerAPI.setDWObject(DWObject);
+    expect(DWObject.IfDuplexEnabled).toEqual(false); // default
+
+    await scannerAPI.startScanSession({
+      applicationContext,
+      scanMode: SCAN_MODES.DUPLEX,
+    });
+    expect(DWObject.IfDuplexEnabled).toEqual(true);
+
+    await scannerAPI.startScanSession({
+      applicationContext,
+      scanMode: SCAN_MODES.FLATBED,
+    });
+    expect(DWObject.IfDuplexEnabled).toEqual(false);
+  });
+
+  it('can enable flatbed scanning by calling startScanSession with scanMode set to `FLATBED`', async () => {
```

Comment:
Why is the feeder involved in this test?

Reply:
When the feeder is enabled, the flatbed is not, and when the flatbed is enabled, the feeder is not. (this applies to Amy's scanner, further testing will be needed for the scanners at the court)

Reply:
Ah-ha — thanks!